src/jobs/index.md

```diff
@@ -11,3 +15,4 @@ The following jobs are open:
 * [Android Engineer]({{site.url}}/jobs/android)
 * [Android Technical Lead]({{site.url}}/jobs/android_tl)
 * [iOS Engineer]({{site.url}}/jobs/ios)
```

Comment:
Should these pages also be removed?

Reply:
It doesn't really matter. Once this PR lands, you can't get to them anymore, thanks to the redirect.